modules/ROOT/pages/dw-core-functions-reduce.adoc

```diff
@@ -19,13 +19,13 @@ accumulator, a null value is returned.
 |===
 | Name   | Description
 | `element` | The current element from the array. Can also be referenced as `&#36;`. If this parameter is not named in the lambda expression.
-| `acc` | The accumulator. Can also be referenced as `&#36;&#36;`. Used to store the result of the lambda expression after each iteration of the reduce operation. 
+| `acc` a| The accumulator. Can also be referenced as `&#36;&#36;`. Used to store the result of the lambda expression after each iteration of the reduce operation.
 
-The accumulator parameter can be set to an intial value using the syntax `acc = initValue`. In this case the lambda expression is called with the first element of the input array, then the result is set as the new accumulator value. 
+The accumulator parameter can be set to an intial value using the syntax `acc = initValue`. In this case the lambda expression is called with the first element of the input array, then the result is set as the new accumulator value.
 
 If an initial value for the accumulator is not set, the accumulator is set to the first element of the input array, and then the lamba expression is called with the second element of the input array.
 
-The inital value of the accumulator and the lambda expression dictate the type of result produced by the reduce function. If the accumulator is set to `acc = {}`, then usually the result will be of type Object. If the accumulator is set to `acc = []`, then usually the result will be of type Array. If the accumulator is set to `acc = ""`, then the result will usually be a String. 
+The inital value of the accumulator and the lambda expression dictate the type of result produced by the reduce function. If the accumulator is set to `acc = {}`, then usually the result will be of type Object. If the accumulator is set to `acc = []`, then usually the result will be of type Array. If the accumulator is set to `acc = ""`, then the result will usually be a String.
```

Comment:
Inconsistent wording in the sentences. Use either " "then usually the result will be..." or "then the result will usually be..."

Reply:
reworded